packages/rich-text/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I'm not sure but it feels like this Babel config should be in the folder which generates code for the tests to avoid some accidental overrides when building packages or Gutenberg app.

Reply:
I couldn't get it to work otherwise. Do you have any ideas why? I'm also struggling with getting it to work on Travis. :)

Reply:
We had to do something similar for the playground:
https://github.com/WordPress/gutenberg/blob/master/playground/.babelrc

Parcel does some magic to the Babel config behind the scenes which forces us to add this Babel config 🤷‍♂ I guess we must have it unless they fix it in Parcel 2.x. We should unless ensure it doesn't get activated for the codebase when executed outside of Parcel.